src/jit/gentree.h

```diff
@@ -687,15 +681,7 @@ struct GenTree
     #define GTF_DONT_CSE        0x00004000  // don't bother CSE'ing this expr
     #define GTF_COLON_COND      0x00008000  // this node is conditionally executed (part of ? :)
 
-#if defined(DEBUG) && SMALL_TREE_NODES
-    #define GTF_NODE_LARGE      0x00010000
-    #define GTF_NODE_SMALL      0x00020000
-
-    // Property of the node itself, not the gtOper
-    #define GTF_NODE_MASK       (GTF_COLON_COND | GTF_MORPHED   | GTF_NODE_SMALL | GTF_NODE_LARGE )
```

Comment:
Something bugs me about this. `GTF_NODE_MASK` is used to prevent copying these flags to other nodes but it contains only 3 of the 4 moved flags. I suspect that some places that deal with flag copying might need to be updated. For example, at the end of `gtClone` there is:
```
copy->gtFlags |= tree->gtFlags & ~GTF_NODE_MASK;
```
Shouldn't the following line be added
```
copy->gtDebugFlags |= tree->gtDebugFlags & ~(GTF_DEBUG_MORPHED | GTF_DEBUG_NODE_SMALL | GTF_DEBUG_NODE_LARGE);
```
to copy `GTF_DEBUG_VAR_CSE_REF`?

Though it's not clear why `GTFD_VAR_CSE_REF` would be copied in this case, maybe the existing code is wrong to begin with.